src/main/java/networkbook/MainApp.java

```diff
@@ -88,6 +89,10 @@ private Model initModelManager(Storage storage, ReadOnlyUserPrefs userPrefs) {
             logger.warning("Data file at " + storage.getNetworkBookFilePath() + " could not be loaded."
                     + " Will be starting with an empty NetworkBook.");
             initialData = new NetworkBook();
+        } catch (NullValueException e) {
+            logger.warning(e.getMessage());
+            logger.warning("Starting with an empty NetworkBook.");
+            initialData = new NetworkBook();
```

Comment:
I feel we can set the output text of the command box to notify user that (1) data file is invalid, (2) reason why it's invalid (basically `e.getMessage()`), (3) app is initialized with empty data which will overwrite original data file

Reply:
This is feature change though.

Reply:
Right, KIV for future development.